app/views/docs/graphql.phtml

```diff
@@ -0,0 +1,497 @@
+        <td data-title="Description: ">
+            The GraphQL query to execute.
+        </td>
+    </tr>
+    <tr>
+        <td data-title="Name: ">
+            operationName
+        </td>
+        <td data-title="">
+            <span class="tag">optional</span>
+        </td>
+        <td data-title="Type: ">
+            string
+        </td>
+        <td data-title="Description: ">
+            If the query contains several named operations, controls which one to execute.
+        </td>
+    </tr>
+    <tr>
+        <td data-title="Name: ">
+            variables
+        </td>
+        <td data-title="">
+            <span class="tag">optional</span>
+        </td>
+        <td data-title="Type: ">
+            object
+        </td>
+        <td data-title="Description: ">
+            An object containing variable names and values for the query. Variables are made available to your query with the <code>$</code> prefix.
+        </td>
+    </tr>
+    </tbody>
+</table>
+
+<h3><a href="/docs/graphql#get-requests" id="get-requests">GET Requests</a></h3>
+
+<p>You can execute a GraphQL query via a GET request, passing a <b>query</b> and optionally an <b>operationName</b> and <b>variables</b> as query parameters.</p>
+
+<h3><a href="/docs/graphql#post-requests" id="post-requests">POST Requests</a></h3>
+
+<p>There are multiple ways to make a GraphQL POST request, differentiated by content type.</p>
+
+<h4>JSON</h4>
+
+<p>There are two ways to make requests with the <b>application/json</b> content type. You can send a JSON object containing a <b>query</b> and optionally an <b>operationName</b> and <b>variables</b>, or an array of objects with the same structure.</p>
+
+<ul class="phases clear" data-ui-phases>
+    <li>
+        <h3>Object</h3>
+        <div class="ide margin-bottom" data-lang="json" data-lang-label="JSON">
+            <pre class="line-numbers"><code class="prism language-json" data-prism>{
+    "query": "",
+    "operationName": "",
+    "variables": {}
+}</code></pre>
+        </div>
+    </li>
+    <li>
+        <h3>Array</h3>
+        <div class="ide margin-bottom" data-lang="json" data-lang-label="JSON">
+            <pre class="line-numbers"><code class="prism language-json" data-prism>[
+    {
+        "query": "",
+        "operationName": "",
+        "variables": {}
+    }
+]</code></pre>
+        </div>
+    </li>
+</ul>
+
+<h4>GraphQL</h4>
+
+<p>The <b>application/graphql</b> content type can be used to send a query as the raw POST body.</p>
+
+<div class="ide margin-bottom" data-lang="graphql" data-lang-label="GraphQL">
+    <pre class="line-numbers"><code class="prism language-graphql" data-prism>query GetAccount {
+    accountGet {
+        email
+    }
+}</code></pre></div>
+
+<h4>Multipart Form Data</h4>
+
+<p>The <b>multipart/form-data</b> content type can be used to upload files via GraphQL. In this case, the form data must include the following parts in addition to the files to upload:</p>
```

Comment:
This is also a general comment, but when highlighting verbatim text, like content type, variables, method names, etc. Use `<code>` for highlighting instead of `<b>`
```suggestion
<p>The <code>multipart/form-data</code> content type can be used to upload files via GraphQL. In this case, the form data must include the following parts in addition to the files to upload:</p>
```